scripts/build_workflow_release.sh

```diff
@@ -42,14 +41,17 @@ function deploy_options() {
   local base_options
   if [ -f ${wdl_dir}/${prefix}.options.json ]; then
     base_options=${prefix}.options.json
+    cp ${wdl_dir}/${base_options} ${target_dir}/${versioned_options}
   elif [ -f ${wdl_dir}/${prefix}.${env}.options.json ]; then
     base_options=${prefix}.${env}.options.json
+    cp ${wdl_dir}/${base_options} ${target_dir}/${versioned_options}
+  elif [ -f ${DSDE_PIPELINES_ROOT}/tests/skylab/test.options.json ]; then
+    cp ${DSDE_PIPELINES_ROOT}/tests/skylab/test.options.json ${target_dir}/${versioned_options}
```

Comment:
Maybe we should change this to check if the wdl directory path includes `skylab` so we don't use these options for other pipelines? 
Also we should be able to just set the `base_options` and then do the copy once. 
```suggestion
    base_options=${DSDE_PIPELINES_ROOT}/tests/skylab/test.options.json
```

Reply:
Done!